src/_includes/layouts/base.njk

```diff
@@ -1,5 +1,5 @@
 <!doctype html>
-<html lang="{{ locale or 'en-ca' }}" class="no-js">
+<html lang="{{ locale or 'en-CA' }}" dir="{{ config.languages[locale].dir }}" class="no-js">
```

Comment:
I think on pages where no locale is set this will fail because `{{ config.languages[locale].dir }}` will be trying to look up a null key for the locale variable. You can do this in Nunjucks instead (reference: https://mozilla.github.io/nunjucks/templating.html#if-expression):

```nunjucks
{{ config.languages[locale].dir if locale else 'ltr' }}
```